app/models/miq_request.rb

```diff
@@ -420,11 +420,27 @@ def task_check_on_execute
   def execute
     task_check_on_execute
 
-    deliver_on = nil
     if get_option(:schedule_type) == "schedule"
-      deliver_on = get_option(:schedule_time).utc rescue nil
+      start_time = get_option(:schedule_time).utc rescue nil
+
+      MiqSchedule.create!(
+        :name         => "Request scheduled",
```

Comment:
@gmcculloug you mentioned using the name here and I considered that, but I didn't see `MiqRequest#name` and didn't want to rely on `source.name` being available since source is polymorphic

Reply:
The property to use here would be `MiqRequest#description`

Reply:
Ah okay thanks, done